src/GapicClientTrait.php

```diff
@@ -318,6 +318,20 @@ private function setClientOptions(array $options)
             $clientConfig,
             $options['disableRetries']
         );
+
+        // Edge case: If the client has the gRPC extension installed, but is
+        // a REST-only library, then the grpcVersion header should not be set.
+        if ($this->transport instanceof GrpcFallbackTransport) {
+            $options['grpcVersion'] = phpversion('grpc');
+            $options['restVersion'] = Version::getApiCoreVersion();
+        } else if ($this->transport instanceof GrpcTransport) {
+            $options['grpcVersion'] = phpversion('grpc');
+            unset($options['restVersion']);
+        } else if ($this->transport instanceof RestTransport) {
+            unset($options['grpcVersion']);
+            $options['restVersion'] = Version::getApiCoreVersion();
```

Comment:
It seems a little odd to send the same value as we do for the gax header, do other languages send the same value twice too?

Reply:
The doc says this should be the "the language-specific REST implementation." If there's a better version we should use that's not gax-php, please let me know!

Reply:
This seems to make the most sense then :).